README.md

```diff
@@ -1,14 +1,55 @@
-# `autofeat` library
+# Autofeat
 
-This library contains `sklearn`-compatible linear prediction models with automated feature engineering and selection capabilities.
+**Autofeat** is a Python library that provides `sklearn`- compatible linear prediction models with automated feature engineering and selection capabilities.
```

Comment:
please remove the space in "`sklearn`- compatible"

Reply:
removed the space between "sklearn" and "compatible"